input/fsh/modules/family.fsh

```diff
@@ -69,15 +70,15 @@ Extension: FamilyType
 Id: family-type
 Title: "Family Type Extension"
 Description: "Extension containing Family Type"
-* value[x] only CodeableConcept 
+* value[x] only CodeableConcept
 * valueCodeableConcept ^short = "Describes the 'type' of study family, eg, trio."
 * valueCodeableConcept from ncpi-family-types-vs (extensible)
 
 Extension: Description
 Id: description
-Title: "Study Family Description"
```

Comment:
Assuming you agree with this change, we can reuse this in (research-accesss-policy.fsh)[input/fsh/modules/research-accesss-policy.fsh]